40 Add storage for comet-ml-panels

Sean1572 opened the pull request.

Sean1572 committed:
Add leaderboard panel

kgarwoodsdzwa reviewed (changes requested):
if you could follow google style docstrings and pass flake8 and pylint we can merge!

Sean1572 committed:
Lint leaderboard.py

Sean1572 committed:
Added task filtering

Sean1572 committed:
Fixed sorting

Sean1572 committed:
Make it easier to add tasks to leaderboard

kgarwoodsdzwa reviewed (commented):
Looks good, only thing is to make sure to update the cometml one with this version, i will merge after that. reven though its the same meat of it, its different than the one on comet ml. also i dont know if this is meant to be this way, but currently on the comet leaderboard, for both metrics, the task 'testing_buowset0' evokes an error screen for the leaderboard. Is that a flaw of the leaderboard code or just that task isn't meant to be apart of the leaderboard?

Sean1572 commented:
> Looks good, only thing is to make sure to update the cometml one with this version, i will merge after that. reven though its the same meat of it, its different than the one on comet ml. also i dont know if this is meant to be this way, but currently on the comet leaderboard, for both metrics, the task 'testing_buowset0' evokes an error screen for the leaderboard. Is that a flaw of the leaderboard code or just that task isn't meant to be apart of the leaderboard?

Lets.... talk about the leaderboard and get yalls thoughts on it. When the docstring is in the file on comet-ml, it displays the full docstring which I'm not a big fan of myself. The testing_buowset0 isn't suppose to be a task that is a part of the leaderboard, but ensuring it doesn't crash is something worth fixing

Sean1572 committed:
Remove docstring

CometML has weird behavior with strings, where it displays docstrings, going to email their support line and see what that is about.

The current solution is to turn it into a comment
